docs/api/action-server.rst

```diff
@@ -1,3 +1,6 @@
+:desc: Check out the API docs for open source chatbot framework Rasa\'s
```

Comment:
What's with the backlash `\'`?

Reply:
Ok, for some reason, I was getting a weird error in my browser as soon as I made that change (and it disappeared once I escaped it). So I thought it must be that quote. I'm not able to reproduce it though. Weird.